Resources/ServerInfo/Guidebook/ServerRules/SiliconRules/RuleS8DefaultCrewDefinition.xml

```diff
@@ -1,4 +1,15 @@
 ﻿<Document>
   # Silicon Rule 8 - Your HUD determines who is crew
   Unless a law redefines the definition of crew, then anyone who the HUD indicates to you has a job, including passengers, is a crewmember. You cannot do something that causes someone to not be considered crew, but you can allow someone else to do something that causes someone to not be crew.
+  
+  The following icons are not considered crew:
+  <Box>
+    <GuideEntityEmbed Entity="NoIdIcon" Caption="No ID"/>
+    <GuideEntityEmbed Entity="UnknownIcon" Caption="Unknown"/>
+    <GuideEntityEmbed Entity="VisitorIcon" Caption="Visitor"/>
+    <GuideEntityEmbed Entity="BorgIcon" Caption="Cyborg"/>
+    <GuideEntityEmbed Entity="StationAiIcon" Caption="Station AI"/>
+    <GuideEntityEmbed Entity="SyndicateIcon" Caption="Syndicate"/>
+    <GuideEntityEmbed Entity="ZombieIcon" Caption="Zombie"/>
```

Comment:
The zombie icon should not be included as it is very unlikely to ever happen in round (As far as I know, only Agent ID can create it, which should probably be removed...).

By including this, it could confuse some silicons into thinking that zombies aren't crew (even without the hud icon).

```suggestion
```